src/article.ts

```diff
@@ -79,14 +79,13 @@ export class Article {
   }
 
   public async publishArticle(): Promise<ArticlePublishedStatus> {
-    const body: ArticleApi = {
-      body_markdown: this.readArticleOnDisk(),
```

Comment:
It looks like you're not using `readArticleOnDisk` anymore which is also calling `updateLocalImageLinks` so that's important.